### PR TITLE
release 1.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: puppet-lint-resource_reference_syntax.gemspec [hunk 1]
```diff
@@ -22,6 +22,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rspec-its', '~> 1.0'
   spec.add_development_dependency 'rspec-collection_matchers', '~> 1.0'
-  spec.add_development_dependency 'rake', '~> 0'
```
Comment (ekohl) [inline puppet-lint-resource_reference_syntax.gemspec:25]: Why is this needed? `~> 0` should allow any version.

Comment (bastelfreak) [reply puppet-lint-resource_reference_syntax.gemspec:25]: I guess it conflicts with the rake dependency from the changelog generator:
```
bastelfreak ~/puppet-lint-resource_reference_syntax $ bundle install --path .vendor/
[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set path '.vendor/'`, and stop using this flag
Fetching gem metadata from https://rubygems.org/........
You have requested:
  rake ~> 0

The bundle currently has rake locked at 13.0.1.
Try running `bundle update rake`

If you are updating multiple gems in your Gemfile at once,
try passing them all to `bundle update`
bastelfreak ~/puppet-lint-resource_reference_syntax $ bundle update rake
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Bundler could not find compatible versions for gem "rake":
  In Gemfile:
    rake (~> 0)

    github_changelog_generator was resolved to 1.15.1, which depends on
      rake (>= 10.0)
bastelfreak ~/puppet-lint-resource_reference_syntax $ 
```

File: .travis.yml [hunk 2]
```diff
@@ -3,17 +3,25 @@ sudo: false
...
-- 2.0.0
-- 2.1.5
-- 2.4.2
 matrix:
   fast_finish: true
+  include:
+  - rvm: 2.0.0
+    bundler_args: --without release
+  - rvm: 2.1.5
+    bundler_args: --without release
```
Comment (ekohl) [inline .travis.yml:12]: 2.1.5 will be used to release now.

Comment (bastelfreak) [reply .travis.yml:12]: but we only need the group to generate the changelog, we don't require it witin travis

**Comment by bastelfreak** [comment]
@ekohl I added Ruby 2.5/2.6 and 2.7 to the CI matrix

File: .travis.yml (hunk 2 above)
Comment (ekohl) [reply .travis.yml:12]: Ah right. So why do you use `include` and not just `rvm:` list? IIRC you can set `bundler_args` at the top level as well.

Comment (bastelfreak) [reply .travis.yml:12]: I tried it at multiple levels but could not find the correct one.

Comment (ekohl) [reply .travis.yml:12]: https://github.com/theforeman/puppet-foreman/blob/d33e10c2d157c392b1289215cb192b00d2e30c98/.travis.yml#L81

Comment (bastelfreak) [reply .travis.yml]: updated

**Comment by tuxmea** [comment]
Looks good to me.
